package.json

```diff
@@ -191,6 +191,13 @@
                                 "type": "command",
                                 "command": "workbench.action.terminal.focus"
                             },
+                            {
+                                "key": "$",
+                                "name": "Run Recent Command in Terminal",
```

Comment:
```suggestion
                                "name": "Terminal: Run Recent Command",
```
Let's use the same name VSCode uses. It seems more concise.

Reply:
Btw, I don't know we have a name with the suggested format "XXX: Action"

Reply:
yeah, you are right, but I find it nice. Anyway, resolving this for now 👍